R/check-input-helpers.R

```diff
@@ -0,0 +1,390 @@
+#' @title Check whether an input is an atomic vector of mode 'numeric'
+#'
+#' @description Helper function
+#' @param x input to check
+#' @param x additional arguments to pass to `check_numeric()`
+#' @importFrom checkmate check_atomic_vector check_numeric
+#' @return Either TRUE if the test is successful or a string with an error
+#' message
+#' @keywords internal
+check_numeric_vector <- function(x, ...) {
+  # check functions must return TRUE on success
```

Comment:
This seeems like it should be in the function documentation not in the comments

Reply:
Don't know where that happened, but the comments have been removed in the meantime. Necessity for documentation has been flagged in #357